.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -214,6 +214,7 @@ jobs:
           endpoint-routes: ${{ matrix.endpoint-routes }}
           ipv4: ${{ matrix.ipv4 }}
           ipv6: ${{ matrix.ipv6 }}
+          underlay: ${{ matrix.underlay }}
```

Comment:
This would also need to live in `tests-e2e-upgrade`, right?

(I really need to find a few moments to rip out the remaining bits of `tests-ipsec-upgrade`)